packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -234,6 +248,9 @@ class DescopeWc extends BaseDescopeWc {
             backupCallbackUri: redirectAuthBackupCallbackUri,
           }
         : undefined;
+    const nativeOptions = native
+      ? { platform: native, nativeOAuthProvider }
```

Comment:
Correct me if I'm wrong- we can have emitter native oauth provider here, no?

Reply:
Not sure what you mean here, let's talk IRL so I can understand